### PR TITLE
Implemented cheap-ruler definition

### PR DESCRIPTION
**Pull request by DenisCarriere**
case 1. Add a new type definition.
- [x] checked compilation succeeds with `--target es6` and `--noImplicitAny` options.
- [x] has correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)
- [x] has a [test file](http://definitelytyped.org/guides/contributing.html#tests) with the suffix of  `-tests.ts` or `-tests.tsx`.

Added `cheap-ruler` definition
https://github.com/mapbox/cheap-ruler

### CODE REVIEW

File: cheap-ruler/cheap-ruler.d.ts [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+         * Given two points of the form [longitude, latitude], returns the distance.
+         *
+         * @param {Array<number>} a point [longitude, latitude]
+         * @param {Array<number>} b point [longitude, latitude]
+         * @returns {number} distance
+         * @example
+         * var distance = ruler.distance([30.5, 50.5], [30.51, 50.49]);
+         * //=distance
+         */
+        distance(a: Array<number>, b: Array<number>): number;
```
Comment (vladima) [inline cheap-ruler/cheap-ruler.d.ts]: from the [code](https://github.com/mapbox/cheap-ruler/blob/master/index.js) it seems that `a` and `b` can be more precisely defined as tuples (`[number, number]`). the same is applicable for `bearing`, `destination` and few other methods below. For simplicity you can define type alias for the point type `type Point = [number, number]` and then use it everywhere where array mean point

**Comment by DenisCarriere** [comment]
@vladima I'm having a hard time using `type Point = [number, number]` because in my tests would fail since all my variables have types as `number[]`.

Almost all the time the user will be using `GeoJSON.Feature<GeoJSON.LineString>` or Point/Polygon and using the GeoJSON definition doesn't read the Point `coordinates` as `[number, number]`.

```javascript
const lineString: GeoJSON.Feature<GeoJSON.LineString> = {
  "type": "Feature",
  "properties": {},
  "geometry": {
    "type": "LineString",
    "coordinates": [
      [-77.031669, 38.878605],
      [-77.029609, 38.881946],
      [-77.020339, 38.884084],
      [-77.025661, 38.885821],
      [-77.021884, 38.889563],
      [-77.019824, 38.892368]
    ]
  }
}

// Tests
ruler.along(lineString.geometry.coordinates, 2.5)
// Type=GeoJSON.LineString.coordinates: number[][]
```

**Comment by vladima** [comment]
it was a suggestion, I'm fine with getting this PR as is

**Comment by DenisCarriere** [comment]
@vladima Thanks for your input, I do like having simple types pre-defined like what you suggested with `type Point = [number, number]`.

**Pull request merged**